app/src/main/java/tech/relaycorp/courier/domain/client/CargoCollection.kt

```diff
@@ -49,7 +50,7 @@ class CargoCollection
     }
 
     private suspend fun storeCargo(data: InputStream) =
-        storeMessage.storeCargo(data)
+        storeMessage.storeCargo(Cargo.wrap(data))
```

Comment:
We probably missed this in a previous review, but I think I suggested to rename the `wrap()` method before to `deserialize()`. It's a bit confusing to see references to a method that won't exist in the core lib.